package.json

```diff
@@ -11,7 +11,7 @@
     "src": "src"
   },
   "scripts": {
-    "test": "eslint bin/ src/ test/ && NODE_APP_INSTANCE=test NODE_PATH=./ nyc mocha"
+    "test": "eslint bin/ src/ test/ && NODE_APP_INSTANCE=test NODE_PATH=./ nyc mocha --timeout 10000"
```

Comment:
Let's redefine in test:
```
    static get INDEX_LIMIT_MAX() {
      return 999
    }
```
So max would be 5, then create 21 item. And test `indexAll` this way we don't need to wait for 1000 to be created in test.